packages/react/src/primitives/components.ts

```diff
@@ -11,6 +12,7 @@ export { FieldGroupIcon, FieldGroupIconButton } from './FieldGroupIcon';
 export { Flex } from './Flex';
 export { Grid } from './Grid';
 export { Heading } from './Heading';
+export { HighlightMatch } from './HighlightMatch';
```

Comment:
That was my idea actually. I think it could be useful to users. I'm fine not exporting it if we don't think it is fully baked yet though. 

https://chakra-ui.com/docs/components/highlight

Reply:
I think expose it will make it easy for users who want to customize the option. They can use it to build their highlight parts just like what I have shown on a few examples on the docs without having to write their own